airbyte-integrations/connectors/source-hubspot/Dockerfile

```diff
@@ -34,5 +34,5 @@ COPY source_hubspot ./source_hubspot
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.1.83
+LABEL io.airbyte.version=0.1.84
```

Comment:
Please bump to a major version if you are introducing a backward incomaptible change

Reply:
good point. done